store/models.py

```diff
@@ -9,21 +10,26 @@ class Book(models.Model):
     description = models.TextField(null=True)
     mrp = models.PositiveIntegerField()
     rating = models.FloatField(default=0.0)
-
     class Meta:
         ordering = ('title',)
 
     def __str__(self):
         return f'{self.title} by {self.author}'
 
+class BookRating(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rate = models.FloatField(default=0.0)
+    user = models.ForeignKey(User,null=True,blank=True, on_delete=models.SET_NULL)
```

Comment:
The rating shall be given as an integer - please read proper instructions.
The user should not be null here, and a better option would be to use `on_delete=models.CASCADE`

You could have also used `unique_together` META option here.